### PR TITLE
Change zone number by clicking at arrow icon

### PR DESCRIPTION
**Pull request by NesChaiyapon**
I added a feature `change phone number by clicking at arrow icons` according to this issue https://github.com/codeforthailand/election-live/issues/145.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -231,20 +232,21 @@ function ZoneView({ provinceId, zoneNo }) {
...
+              <Arrow
+                onLeftArrowClick={() => {
+                  navigate(zonePath({ provinceId, no: zoneNo - 1 || 1 }))
                 }}
-              />
+                onRightArrowClick={() =>
+                  navigate(
+                    zonePath({
+                      provinceId,
+                      no: zoneNo < province.zone ? zoneNo + 1 : province.zone,
```
Comment (p16i) [inline src/pages/index.js:243]: For these two boundary cases, what do you think if we hide the corresponding arrow or do a circular visit?

Comment (NesChaiyapon) [reply src/pages/index.js:243]: @heytitle  I think it would be nice if we hide the corresponding arrow, I will hide it.

Comment (p16i) [reply src/pages/index.js:243]: thanks!

**Comment by p16i** [comment]
LGTM.

BTW, as LeftArrow and RightArrow are almost the same, maybe we can have only one Arrow component and use set rotation degree as a prop accordingly.

**Comment by NesChaiyapon** [comment]
Thank you for your suggestion, I will improve the arrow and make it easier to click.

**Pull request merged**